flowzone.yml

```diff
@@ -188,7 +206,45 @@ on:
       CUSTOM_JOB_SECRET_3:
         description: "Optional secret for using with custom jobs"
         required: false
+
     inputs:
+      app_id:
+        description: "GitHub App id to impersonate"
+        type: string
+        required: false
+        # https://github.com/organizations/product-os/settings/apps/flowzone-app
+        # https://github.com/organizations/product-os/settings/variables/actions/APP_ID
+        default: "${{ vars.APP_ID || 291899 }}"
+      # not needed if installed on this current org/repo
+      installation_id:
+        description: "GitHub App installation id"
+        type: string
+        required: false
+        # https://github.com/organizations/product-os/settings/installations
+        # https://github.com/organizations/product-os/settings/variables/actions/INSTALLATION_ID
+        default: "${{ vars.INSTALLATION_ID || 34040165 }}"
+      token_scope:
+        description: "Ephemeral token scope(s)"
+        type: string
+        required: false
+        # https://github.com/organizations/product-os/settings/installations/34040165
+        # https://docs.github.com/en/rest/apps/apps?apiVersion=2022-11-28#create-a-scoped-access-token
+        default: >-
+          {
+            "actions": "read",
+            "administration": "write",
+            "checks": "read",
+            "contents": "write",
+            "members": "read",
+            "metadata": "read",
+            "organization_secrets": "read",
+            "packages": "write",
+            "pages": "write",
+            "pull_requests": "read",
+            "secrets": "read",
+            "statuses": "read",
+            "workflows": "read"
+          }
```

Comment:
Could we require this to be base64 encoded so it's less likely to get mangled in a workflow file?

Reply:
@klutchell I would prefer this to be left as-is so it's clear what the token scope is. Looking at base64 strings isn't the best UX.

Reply:
fair enough